docs/B-Computations/logic.md

```diff
@@ -14,7 +18,7 @@ After reading this section, you will be able to:
 
 A complete programming language includes facilities to implement sequential constructs, in which one statement follows another and the statements are executed in order, and two other constructs, which represent modifications of sequential constructs.  Selection constructs represent different paths through the set of instructions.  Iteration constructs represent repetition of the same set of instructions until a specified condition has been met.  The three classes of constructs required to complete a programming language are illustrated in the figure below.
 
-![](https://ict.senecacollege.ca//~ipc144/pages/images/constructs.png)
+![constructs](https://ict.senecacollege.ca//~ipc144/pages/images/constructs.png)
```

Comment:
Suggestion to provide meaningful alt text. Example: `Diagrams of three classes of constructs`

https://supercooldesign.co.uk/blog/how-to-write-good-alt-text

Reply:
Amazing work!
Thank you @abatomunkuev !